### DISCUSSION TITLE
Advice whether and how to migrate personal spaces

### DISCUSSION THREAD

**Discussion by haroldvandekamp** [WikiTransformationProject/wikitraccs-releases#57] [Q&A] [answered]
During the inventory of a customer's Confluence environment, hundreds of personal spaces have been found. 
What is the advice on that regarding migration? Do you advice users to migrate their relevant content manually by themselves to their personal OneDrive? E.g.:
1. They need to copy over their relevant files
2. They should export (copy over by hand) relevant pages and blogs they created in Word document

Do you think of this approach? Or do you see other valuable migration routes for migrating personal spaces?

**Comment by heinrich-ulbricht** [comment, marked as answer]
@haroldvandekamp I'm afraid I have no definitive answer for that. I have seen the manual approach as well, spreading the workload out to the users, combined with an info banner in Confluence and a deadline. Not sure though about concrete advice on where to move the pages to. Export to Word/PDF to archive in OneDrive? Copy everything to OneNote? Or Loop? Might depend on company guidelines on where to store which kind of information, going forward.

**Reply by haroldvandekamp** [reply]
Thanks for your insight and answer. That's valuable. 

I'm indeed advising them to migrate personal spaces to OneDrive.
